CODE_OF_CONDUCT.md

```diff
@@ -95,71 +104,84 @@ We will not accept harassment or other exclusionary behaviours, such as:
 
-The conda Organization is committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates the conda Organization Code of Conduct, please complete the form below to make a report.
+# Live Events: Immediate and Long Term Actions and Consequences
 
-**REPORTING FORM:** [*https://URL/to/your/CoC/reporting/form*](https://URL/to/your/CoC/reporting/form)
+> **If you feel your safety is in jeopardy or the situation is an emergency, we urge you to contact local law enforcement before making a report to the event's Code of Conduct representatives or other staff.** (In the U.S., call 911.)
 
-Reports are sent to the conda Organization Code of Conduct Enforcement Team (see below).
+Live events present particular challenges: 
+  
+**Code of conduct reports, and consequences that stem from them, merit a thoughtful and deliberative process.  Decisions and consequences matter for the reporter, the reported, and for the community at large.  However, many reports, especially at live events, require rapid action to quickly address the behavior being reported.**
 
-<!-- If you are using TypeForm, leave the following text in place -->
+To better support situations where immediate action may be required, these guidelines are used *during* live events:
 
-You can view the Privacy Policy and Terms of Service for TypeForm at [*https://terms.typeform.com/to/fJAmVc*](https://terms.typeform.com/to/fJAmVc).
+* All conda Organization events will have specific, named Code of Conduct contacts for the events.
+* The names and contact mechanisms for the Code of Conduct representatives will be clearly and frequently communicated to event participants.
 
-#### Person(s) Responsible for Resolving Complaints
+At conda Organization events, Code of Conduct representatives will attempt to gather and write down [information](#what-to-include-in-a-report) from anyone making a verbal report at a live event. Recording the details in writing is exceedingly important in order for us to effectively respond to reports. If event staff write down a report taken verbally, then the person making the report will be asked to review the written report for accuracy.
 
-All reports of breaches of the code of conduct will be investigated and handled by the **conda Organization Code of Conduct Enforcement Team**.
+For reports made during live events, or in any situation where urgent action is needed:
 
-The current conda Organization Code of Conduct Enforcement Team consists of:
+* Any two (or more) event organizers, event staff, CoC Committee members or CoC representatives can decide if immediate action is to be taken and what that action is.  In exceptionally dangerous situations, this decision can be made by a single person.  
```

Comment:
What's a "CoC representatives"?

Reply:
I will try and make that clearer.  The goal is to have easy to reach CoC representative at any conda event, even if there aren't CoC members at that event.  We want people in the room who can react quickly.

Reply:
Ah, so the CoC committee may delegate to one or more representatives for the duration of event, that's smart. Thanks for elaborating!